src/API/Configurations.js

```diff
@@ -15,8 +15,13 @@ export const useListQuery = options => {
     ...options,
   });
 
+  const configurations = orderBy(
+      query.data?.configurations ?? [],
+      ['name'],
+  );
```

Comment:
I guess plain `Array.sort` using [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) as comparator would still play better here:
- minus one `lodash` dependency
- `localeCompare` groups upper and lower case letters together  **by default**. While it's not strictly important here, having it for free is a obvious plus.

`lodash.orderBy` sorting:
![image](https://user-images.githubusercontent.com/72964370/124592693-6526bc80-de66-11eb-9617-04d60256e55d.png)

`localeCompare` variant:
![image](https://user-images.githubusercontent.com/72964370/124592544-41fc0d00-de66-11eb-8f8d-85661a4da8dc.png)

Reply:
It's not an "imperative for a change", but more of a discussion point.